dialog/file.go

```diff
@@ -181,10 +184,36 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 
 	favorites := f.loadFavorites()
 
-	favoritesGroup := widget.NewGroup("Favorites", favorites...)
-	return fyne.NewContainerWithLayout(layout.NewBorderLayout(header, footer, favoritesGroup, nil),
-		favoritesGroup, header, footer, body)
+	favoritesGroup := widget.NewVScrollContainer(widget.NewGroup("Favorites", favorites...))
+	var optionsButton *widget.Button
+	optionsButton = widget.NewButton("Options", func() {
+		f.optionsMenu(fyne.CurrentApp().Driver().AbsolutePositionForObject(optionsButton), optionsButton.Size())
+	})
+
+	left := fyne.NewContainerWithLayout(layout.NewBorderLayout(nil, optionsButton, nil, nil), favoritesGroup, optionsButton)
+
+	return fyne.NewContainerWithLayout(layout.NewBorderLayout(header, footer, left, nil),
+		left, header, footer, body)
+
+}
+
+func (f *fileDialog) optionsMenu(position fyne.Position, buttonSize fyne.Size) {
+	hiddenFiles := widget.NewCheck("Show Hidden Files", func(changed bool) {
+		f.showHidden = changed
+		f.refreshDir(f.dir)
+	})
+	hiddenFiles.SetChecked(f.showHidden)
+
+	title := widget.NewLabel("Options")
+	title.Alignment = fyne.TextAlignCenter
+	title.TextStyle = fyne.TextStyle{Bold: true}
+
+	content := widget.NewVBox(widget.NewCard("", "", title),
```

Comment:
You should use `container.NewVBox` instead, `widget.NewVBox` is deprecated.
As we probably can ignore the label, you can remove the card here.

Reply:
Done